tests/bdd/steps/feature_steps/http.feature

```diff
@@ -0,0 +1,10 @@
+Feature: HTTP steps
+  Scenario: the response is HTML
```

Comment:
These one-step feature steps prompted me to wonder: is it always necessary to have the feature-step abstraction? Would a test also be able to use `the response header 'Content-Type' matches `'^text/html'` directly, without the feature-step "wrapper?"

Reply:
There's no problem with calling the steps directly. We do that in several places, so it's not a rule or anything.

The thinking here was:

 * This version of the step better declares what we are testing, rather than how we are testing it: e.g. having the header alone is probably not enough to test we are really returning HTML, but it's an ok stand in for now.
 * This makes the step easier to maintain. If we decided we didn't like how we were testing for HTML we could change it in one place and have it update everywhere
 * It's quite likely we might also want to test that the body has "<html" or something in it too, and this would give a nice home for it